pkg/connector/connector.go

```diff
@@ -93,12 +93,17 @@ func (o *OneLogin) Validate(ctx context.Context) (annotations.Annotations, error
 func New(ctx context.Context, clientId, clientSecret, subdomain string, syncPrivileges bool) (*OneLogin, error) {
 	httpClient, err := uhttp.NewClient(ctx, uhttp.WithLogger(true, ctxzap.Extract(ctx)))
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("onelogin-connector: failed to create HTTP client for connector: %w", err)
 	}
 
-	oneLoginClient, err := onelogin.NewClient(ctx, httpClient, clientId, clientSecret, subdomain)
+	wrappedClient, err := uhttp.NewBaseHttpClientWithContext(ctx, httpClient)
```

Comment:
it's strange to create the http client outside of the client code. we don't do this elsewhere.